### PR TITLE
Add initial cut at User personas

### PR DESCRIPTION
**Pull request by kevinrobinson**
This came up during Code for Boston in https://github.com/studentinsights/studentinsights/issues/208, and this is partially addressing https://github.com/studentinsights/studentinsights/issues/156.

The background here is that in working on improvements or features that involve some aspect of product or design, it's often helpful to talk about our users as a way to understand what problems they have and how we can help.  This is mostly done in-person, and we're not really capturing any of this in the project in a way that makes it easier for us as a diverse and often distributed team to build a shared understanding here.

Doing this in person is great! and we should keep doing it.  But this is a first step at capturing some of this as a way to help onboard new team members more easily, and as a way to make concrete some of the important background knowledge and assumptions that folks have who've been working on the project longer.

### CODE REVIEW

**Comment by kevinrobinson** [comment]
@erose @johnrobbjr and Dimos, this is what we talked about in person for a bit at Code for Boston tonight.

@alexsoble Please chime in as well!  We've talked about this a bit offline, but this is something I'd love to talk more about when you're back in town. :)

**Comment by kevinrobinson** [comment]
Also, the intent here is that we're more regularly updating here, and as @johnrobbjr suggested immediately, that folks are prompting me to write this information down as it comes up in issues or in the moment, or freely opening new issues asking for clarification or more background.  Essentially we should approach talking about this and writing about it just like we do with talking about code and writing it. :)

**Comment by alexsoble** [comment]
@kevinrobinson @erose @johnrobbjr 

I would say these user personas look fantastic!

Two thoughts:

+ We could send these to Jill and Uri (two of our users) to hear what they think of these descriptions and what they would add.

+ A small, cute, and useful Code for Boston design / art task might be creating little illustrations for each of these personae.

**Comment by kevinrobinson** [comment]
Yes, that would be awesome! We definitely need more work here, and those
are great next steps.

**Comment by alexsoble** [comment]
@kevinrobinson School is out today b/c of Good Friday so I will email Uri & Jill on Monday. :+1:

File: README.md [hunk 1]
```diff
@@ -9,7 +9,16 @@ Check out the [demo site](https://somerville-teacher-tool-demo.herokuapp.com/):
   - username: `demo@example.com`
   - password: `demo-password`
 
-## Product overview
+## User personas: Who we're serving
+There are three main user personas we're serving. Principals, Interventionists and Classroom Teachers. Right now we're focused primarily on serving principals, and the rough progression will likely be to Interventionists next. Early adopter Classroom Teachers are great, but focused on scaling adoption across all classroom teachers isn't a priority yet.
+
+Principals are responsible for a school, from ensuring all students are progressing academically to making hiring and staffing decisions for teachers.
+
+Interventionists are typically folks who provide some kind of specialized service to students, like counseling, behavioral services or specialized reading instruction. They caseloads of 20-70 students and are often involved in interdisciplinary teams focused on supporting students who are most at-risk.
```
Comment (eudaimonious) [inline README.md]: Typo "They caseloads"

Comment (kevinrobinson) [inline README.md]: fixed, thanks!

**Comment by kevinrobinson** [comment]
@alexsoble Sounds great!  I added https://github.com/studentinsights/studentinsights/issues/247 for your great suggestion of images.  Shall we merge this, and then we update based on feedback from our school amigos?

**Comment by alexsoble** [comment]
@kevinrobinson Yep, merging!  😊

**Pull request merged**